### PR TITLE
Added Left Click Option for Kill Aura

### LINKED ISSUE #363: [Suggestion] Fish movement module
**Describe the feature**
Addition of the fish module.
It basically disables underwater gravity, preventing the player from sinking (like in creative for example).
The player is able to go down by pressing the crouch button and go up by pressing the spacebar.

**Screenshots**
2021-03-28 19_39_02-Minecraft_ 1 16 5 - Singleplayer

**Additional info**
Source code from Wurst's version of the module:
https://github.com/Wurst-Imperium/Wurst7/blob/a1adc128622fe6f8162e1fd922db6706b9b44c99/src/main/java/net/wurstclient/hacks/FishHack.java

### LINKED ISSUE #636: [SUGGESTION] Kill aura only on left click
**Describe the feature**
ClickAura, like killaura, but only when you click.

### PR DESCRIPTION
**Pull request by StormyBytes**
Resolves #636

### CODE REVIEW

**Comment by seasnail8169** [comment]
isn't this what trigger is?

**Comment by StormyBytes** [comment]
Trigger attacks only when you look at the entity

**Comment by MobESP** [comment]
What is this point of this?

**Comment by Declipsonator** [comment]
> What is the point of this?

so that ka only attacks when you want it to

**Comment by seasnail8169** [comment]
can we merge trigger and this into KA as one? imo that would make more sense since trigger is literally just ka when looking at someone

**Comment by ikeafridge4** [comment]
maybe add this as a setting in trigger

**Comment by Declipsonator** [comment]
> can we merge trigger and this into KA as one? imo that would make more sense since trigger is literally just ka when looking at someone

Yeh, we could probably just put a only when looking option

File: src/main/java/minegame159/meteorclient/systems/modules/combat/KillAura.java [hunk 1]
```diff
@@ -63,6 +63,20 @@
             .build()
     );
 
+    private final Setting<Boolean> leftclick = sgGeneral.add(new BoolSetting.Builder()
+            .name("left-click")
```
Comment (seasnail8169) [inline src/main/java/minegame159/meteorclient/systems/modules/combat/KillAura.java:67]: Could the naming be consistent please? Maybe "only on click"

**Review by seasnail8169 — CHANGES_REQUESTED** [review]

**Pull request merged**